src/backend/distributed/sql/udfs/worker_split_copy/11.1-1.sql

```diff
@@ -0,0 +1,17 @@
+DROP TYPE IF EXISTS citus.split_copy_info;
+CREATE TYPE citus.split_copy_info AS (
+    destination_shard_id bigint,
+    destination_shard_min_value text,
+    destination_shard_max_value text,
+    -- A 'nodeId' is a uint32 in CITUS [1, 4294967296] but postgres does not have unsigned type support.
+    -- Use integer (consistent with other previously defined UDFs that take nodeId as integer) as for all practical purposes it is big enough.
+    destination_shard_node_id integer);
+
+CREATE OR REPLACE FUNCTION pg_catalog.worker_split_copy(
```

Comment:
There seems to be a minor issue with non-superuser owned tables:

```SQL
CREATE ROLE test_role WITH LOGIN;
SET ROLE test_role;


SET citus.next_shard_id TO 897000;
SET citus.next_placement_id TO 877000;
SET citus.shard_count TO 4;
SET citus.shard_replication_factor TO 1;

CREATE TABLE test_table(a int);
SELECT create_distributed_table('test_table', 'a');


SELECT citus_split_shard_by_split_points(897002, ARRAY['10000'], ARRAY[4, 4]);
ERROR:  permission denied for schema citus
CONTEXT:  while executing command on localhost:9702
```

I think we *might* need `GRANT USAGE ON SCHEMA citus TO public;` at some point. Marco might have a better idea, but I think it is right after creating `split_copy_info`

Also, it is always useful to add one regression test without superuser (by default the regression tests run with superuser -- other than arbitrary configs tests)

Reply:
> I think we might need GRANT USAGE ON SCHEMA citus TO public; at some point. 

that would recreate the "every table gets created in the citus schema on Marlin by default" debacle.

Probably split_copy_info should be in pg_catalog rather than citus, though we don't run into this issue in other cases.

Reply:
Thanks @onderkalaci for pointing this out. I have made the fix Marco suggested and updated the test to verify this scenario.  Will keep in mind to have one regression test without superuser in future. 

Resolving this. Please free to reopen if you have any more concerns.